docs/01/b.md

```diff
@@ -2,13 +2,13 @@
   [algebra]: https://github.com/non/algebra
   [Equivalence]: http://en.wikipedia.org/wiki/Equivalence_relation
 
-## Eq
+## `Eq`
```

Comment:
Yea the toc gets messed up because of a Pamflet bug 

![screnshot](http://s13.postimg.org/vkcpm1kef/messedup_toc.png)